docs/user-guide/plot-types/mesh3d-plot.ipynb

```diff
@@ -0,0 +1,117 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0",
+   "metadata": {},
+   "source": [
+    "# 3D mesh plot\n",
+    "\n",
+    "This notebook illustrates how to render 3D meshes,\n",
+    "by supplying a list of vertex positions and vertex indices to construct the mesh faces."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import scipp as sc\n",
+    "import plopp as pp\n",
+    "from plopp.data import examples"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2",
+   "metadata": {},
+   "source": [
+    "## Loading mesh data\n",
+    "\n",
+    "We load a file containing the data to construct the [Utah teapot](https://en.wikipedia.org/wiki/Utah_teapot)."
```

Comment:
Can you explain the data format? It is not obvious to me how to encode vertices and faces.